Makefile

```diff
@@ -34,7 +34,7 @@ ifndef PUBLISH
 override IMAGE_VERSION := $(IMAGE_VERSION)-$(GIT_COMMIT)
 endif
 endif
-IMAGE_TAG ?= $(REGISTRY)/$(IMAGE_NAME):$(IMAGE_VERSION)
+CSI_IMAGE_TAG ?= $(REGISTRY)/$(IMAGE_NAME):$(IMAGE_VERSION)
 IMAGE_TAG_LATEST = $(REGISTRY)/$(IMAGE_NAME):latest
```

Comment:
Should we also rename `IMAGE_TAG_LATEST` for consistency, i.e. `CSI_IMAGE_TAG_LATEST`?

Reply:
Sure. Modified it just now.